docs/UserGuide.md

```diff
@@ -300,33 +300,35 @@ by adding your first Delivery
 
 2. After registering for an account, you should be greeted by some sample Customers
    that have been generated by HomeBoss.
-3. Let's assume that a new customer, Gabriel, is to be added, using the `customer add`{.swift} command.
 
-> `NAME`: Gabriel
+3. Let's assume that a new Customer, Alex, is to be added, using the `customer add`{.swift} command.
+
+> `NAME`: Alex
```

Comment:
Would it be better to keep the name as Gabriel as the values were specifically chosen to match the screenshot

Reply:
yep didnt realise thx